### ISSUE TITLE
is this model any good?

### ISSUE THREAD

**Issue by ren85** [jinglescode/time-series-forecasting-pytorch#6]
If I were to build a model that predicts yesterdays closing price as today's closing price we would also get similar graph, but the model is useless.  Where are prediction accuracy numbers? I bet they would be terrible.

**Comment by jinglescode** [comment]
Hi @ren85,
As we are prediction values, we used MSE. You can check the MSE from the logs, for example, on the training and validation set, where `loss` is the MSE:
```
Epoch[100/100] | loss train:0.006102, test:0.000972 | lr:0.000100
```

**Comment by ren85** [comment]
The question is is this any better than using yesterdays closing price as today's prediction

**Comment by jinglescode** [comment]
> is this any better than using yesterdays closing price as today's prediction

To find that out, what you can do is to calculate the error using MSE of `yesterdays closing price and today's prediction`, and then compare it against the MSE of the model after training.
